### PR TITLE
add Filipa Carvaho as co-author

### PR DESCRIPTION
**Pull request by soerenthomsen**
@pipa85 is this fine for you? Please check and update your contributions accordingly.

### CODE REVIEW

**Comment by patricialg** [comment]
Great :)

**Review by pipa85 — APPROVED** [review]
I'm also chasing after the deep (4km deep) glider oxygen integration.

**Review by pipa85 — CHANGES_REQUESTED** [review]
I only now noticed that my last name is misspelled. 

It's Filipa Carvalho. #170 
Tried editing it, but that option is in greyed out...
still getting the hang of GitHub ...

File: sections/authors_SOP_development_process.md [hunk 1]
```diff
@@ -24,6 +24,7 @@
   2. [Evi Bourma](https://github.com/evibourma), *Hellenic Centre for Marine Research (HCMR)/Institute of Oceanography, Athens, Greece*
   1. [Clark Richards](https://github.com/richardsc), *Fisheries and Oceans Canada, Bedford Institute of Oceanography, Halifax, Canada*
   1. [Stuart Pearce](https://github.com/s-pearce), *Ocean Observatories Initiative, Endurance Array, Oregon State University, Oregon, USA*
+  1. [Filipa Carvaho](https://github.com/pipa85), *National Oceanography Centre, Southampton, UK*
```
Comment (pipa85) [inline sections/authors_SOP_development_process.md:27]: [Filipa Carvalho](https://github.com/pipa85), *National Oceanography Centre, Southampton, UK*

File: sections/authors_SOP_development_process.md [hunk 2]
```diff
@@ -120,3 +121,4 @@ He reviewed the overall document and provided feedback on the overall document.
 
 [Stuart Pearce](https://github.com/s-pearce) provided input to the Aanderaa optode section related to MK2 equation, SVU/Multipoint-calibration and firmwares. 
 
+[Filipa Carvaho](https://github.com/pipa85) provided feedback on overall document and helped to improve the overview schematic.
```
Comment (pipa85) [inline sections/authors_SOP_development_process.md:124]: [Filipa Carvalho](https://github.com/pipa85) provided feedback on overall document and helped to improve the overview schematic.

**Review by pipa85 — CHANGES_REQUESTED** [review]
I think I'm fixing the name ... sorry GitHub is still not my friend

File: sections/authors_SOP_development_process.md (hunk 2 above)
Comment (soerenthomsen) [reply sections/authors_SOP_development_process.md]: Sorry for this!

**Comment by soerenthomsen** [comment]
Thanks for checking! It might also be related to permissions to edit. I have to check!

**Comment by soerenthomsen** [comment]
I fixed the names at all 3 places! Sorry for this!

**Review by patricialg — APPROVED** [review]
Perfect! Thanks Soeren :)

**Pull request merged**